portality/forms/application_processors.py

```diff
@@ -447,6 +447,16 @@ def _send_application_approved_email(self, journal_title, publisher_name, email,
             self.add_alert('Sending the journal acceptance information email didn\'t work. Please quote this magic number when reporting the issue: ' + magic + ' . Thank you!')
             app.logger.exception('Error sending application approved email failed - ' + magic)
 
+    def validate(self):
+        _statuses_not_requiring_validation = ['rejected', 'pending', 'in progress', 'on hold']
```

Comment:
Usually we'd use `constants.APPLICATION_STATUS_REJECTED` etc. but I don't think we use a js equivalent, this is no worse!